.github/workflows/playwright.yml

```
@@ -0,0 +1,30 @@
+name: Playwright Tests
+on:
+  push:
+    branches: [main]
+  pull_request:
+    branches: [main]
```

I meant changing only `pull_request` to main. `push` could remain for dev.

```suggestion
  push:
    branches: [dev]
  pull_request:
    branches: [main]
```